stories/module-adsense-components.stories.js

```diff
@@ -24,30 +24,130 @@ import DashboardSummaryWidget from '../assets/js/modules/adsense/components/dash
-	dashboardSummaryWidget28DailyData,
-	dashboardSummaryWidgetTodayOptions,
-	dashboardSummaryWidgetPeriodOptions,
-	dashboardSummaryWidget28DailyOptions,
-	dashboardTopEarningPageWidgetData,
-	dashboardTopEarningPageWidgetOptions,
-} from '../assets/js/modules/adsense/datastore/__fixtures__';
 
 generateReportBasedWidgetStories( {
 	moduleSlugs: [ 'adsense' ],
 	datastore: STORE_NAME,
 	group: 'AdSense Module/Components/Dashboard/Summary Widget',
+	referenceDate: '2020-09-12',
 	data: [
-		dashboardSummaryWidgetTodayData,
-		dashboardSummaryWidgetPeriodData,
-		dashboardSummaryWidget28DailyData,
+		{
+			kind: 'adsense#report',
+			totalMatchedRows: '1',
+			headers: [
+				{
+					name: 'EARNINGS',
+					type: 'METRIC_CURRENCY',
+					currency: 'USD',
+				},
+				{
+					name: 'PAGE_VIEWS_RPM',
+					type: 'METRIC_CURRENCY',
+					currency: 'USD',
+				},
+				{
+					name: 'IMPRESSIONS',
+					type: 'METRIC_TALLY',
+				},
+			],
+			rows: [
+				[
+					'6.13',
+					'2.98',
+					'226791',
+				],
+			],
+			totals: [
+				'6.13',
+				'2.98',
+				'226791',
+			],
+			averages: [
+				'6.13',
+				null,
+				'226791',
+			],
+			startDate: '2020-09-12',
+			endDate: '2020-09-12',
+		},
+		{
+			kind: 'adsense#report',
+			totalMatchedRows: '1',
+			headers: [
+				{ name: 'EARNINGS', type: 'METRIC_CURRENCY', currency: 'USD' },
+				{ name: 'PAGE_VIEWS_RPM', type: 'METRIC_CURRENCY', currency: 'USD' },
+				{ name: 'IMPRESSIONS', type: 'METRIC_TALLY' },
+			],
+			rows: [
+				[ '2103.23', '1.89', '5693093' ],
+			],
+			totals: [
+				'2103.23', '1.89', '5693093' ],
+			averages: [
+				'2103.23', null, '5693093',
+			],
+			startDate: '2020-08-15',
+			endDate: '2020-09-11',
+		},
+		{
+			kind: 'adsense#report',
+			totalMatchedRows: '32',
+			headers: [
+				{ name: 'DATE', type: 'DIMENSION' },
+				{ name: 'EARNINGS', type: 'METRIC_CURRENCY', currency: 'USD' },
+				{ name: 'PAGE_VIEWS_RPM', type: 'METRIC_CURRENCY', currency: 'USD' },
+				{ name: 'IMPRESSIONS', type: 'METRIC_TALLY' },
+			],
+			rows: [
+				[ '2020-09-01', '18.08', '3.98', '1278373' ],
+				[ '2020-09-02', '27.02', '3.39', '1307556' ],
+				[ '2020-09-03', '35.05', '3.69', '1491923' ],
+				[ '2020-09-04', '45.43', '3.72', '1244807' ],
+				[ '2020-09-05', '44.44', '4.36', '971017' ],
+				[ '2020-09-06', '40.37', '3.77', '1014415' ],
+				[ '2020-09-07', '36.09', '3.35', '1509543' ],
+				[ '2020-09-08', '22.19', '2.96', '1591345' ],
+				[ '2020-09-09', '25.53', '3.11', '1494963' ],
+				[ '2020-09-10', '18.80', '3.23', '1249817' ],
+				[ '2020-09-11', '5.77', '2.88', '330158' ],
+				[ '2020-09-12', '6.13', '2.98', '226791' ],
+			],
+			totals: [ '', '324.90', '3.45', '12465901' ],
+			averages: [ '', '27.08', null, '1038825' ],
+			startDate: '2020-09-01',
+			endDate: '2020-09-12',
+		},
 	],
 	options: [
-		dashboardSummaryWidgetTodayOptions,
-		dashboardSummaryWidgetPeriodOptions,
-		dashboardSummaryWidget28DailyOptions,
+		{
+			dateRange: 'today',
```

Comment:
This should be replaced with start and end dates, correct?

Reply:
I didn't wanna replace these just yet since they rely on special AdSense logic we don't have elsewhere. `today` would be trivial to do, but e.g. `this-month` would require another function or some extra logic in the existing function. I'd rather do that via a separate follow-up issue.

Reply:
@aaemnnosttv I've opened #2477 for the above.